test/test_image.py

```diff
@@ -403,8 +389,6 @@ def test_encode_jpeg_windows(img_path):
         assert_equal(jpeg_bytes, pil_bytes)
 
 
-@cpu_only
-@_collect_if(cond=IS_WINDOWS)
```

Comment:
I removed the `dont_collect` mark so now:

- `test_write_jpeg_windows` is still run on linux and macos, which is fine since it passes on these OSs.
- `test_write_jpeg` is now **skipped** on windows instead of being ignored / not collected. This isn't an issue, as fbcode doesn't run on windows anyway, so skipping is fine.

If this ever becomes an issue again, we can introduce the `dont_collect` mark back, but I'd prefer keeping it away for now.

Reply:
I think keep calling it `test_write_jpeg_windows` after this change can lead to confusion. I would recommend renaming it.